.github/workflows/python.yml

```diff
@@ -74,15 +69,15 @@ jobs:
           docker-compose push ubuntu-python
 
   conda:
+    # this is the nopandas build
     name: AMD64 Conda Python ${{ matrix.python }}
```

Comment:
I think a "nopandas" build needs to be clearly marked as such since it is the exceptional case, not the typical one. I would guess than over 90% of pyarrow installs also have pandas installed.

There are hundreds of skipped tests when pandas is not installed so if someone is looking at one of these nopandas builds that might be misled like I was in the other PR